### DISCUSSION TITLE
[BUG] Data base: No se está viendo los archivos adjuntos con sólo los permisos db-properties-write, admin-properties-write

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#217] [4. Bug Reports] [answered]
Hola

Me di cuenta que si un usuario sólo tiene los permisos: **db-properties-write**, **admin-properties-write** puede ver la información en database, pero puede ver algún archivo adjunto.

Adjunto dos imagenes

Usuario con sólo permisos: **db-properties-write, admin-properties-write**
![image](https://user-images.githubusercontent.com/61419471/119895636-03407200-bf0c-11eb-8ece-ecc4450e72b9.png)

Usuario con los permisos: **db-properties-read, admin-properties-read, db-properties-write, admin-properties-write, read admin,admin-*-write,web-admin-read,web-admin-write,modify-permissions,web-survey-read,admin-*-read,web-access**
![image](https://user-images.githubusercontent.com/61419471/119895831-371b9780-bf0c-11eb-8378-03ab9142324d.png)

**Comment by gmoplaza** [comment]
Hola @NicolasPLG efectivamente parece ser un bug, lo voy a reportar, muchas gracias!.

**Reply by gmoplaza** [reply]
Aprovecho de recordarte que no es necesario el uso de ambos permisos _db-properties-write_, _admin-properties-write,_ solo con **db-properties-write** deberìa ser suficiente para realizar todas las tareas de lectura y escritura en DataBase, si en algùn momento no es asi nos avisas.
Saludos!

**Reply by NicolasPLG** [reply]
hola @gmoplaza si, me habias comentado que no es necesario, sólo los agregué porque me salía que estaba cargando. Igual está reportado internamente.

**Comment by guillevalin** [comment, marked as answer]
@NicolasPLG efectivamente hay un bug, pero no es relacionado con los roles de acceso, les avisaremos cuando esté resuelto. En caso de necesitar priorizarlo por favor contactarte con tu representante de Cotalker.

**Reply by NicolasPLG** [reply]
hola @guillevalin ya está avisado a @jdurandpaz sobre la priorización. 

Cerraré este ticket

Gracias

**Reply by guillevalin** [reply]
@NicolasPLG te cuento que encontramos la razon de este incidente y no es un tema de permisos, de todas maneras te cuento que ya está corregido en productivo. Saludos!
